db/migrate/20210907201954_add_division_to_question_sets.rb

```diff
@@ -0,0 +1,11 @@
+class AddDivisionToQuestionSets < ActiveRecord::Migration[6.1]
+  def change
+    add_reference :question_sets, :division, foreign_key: true, index: true
+    reversible do |dir|
```

Comment:
is there a way to write this so the down process is clearer?

Reply:
I'm not sure what you mean. The down process is trivial since it's just removing a column. I guess we could make separate up and down methods. Would that work?

Reply:
Yes, separate up and down methods would be great. I know Rails added this reversible block, but for future devs reading this (in the unfortunate case we have to revisit this migration) i think it's better to have down and up separate.